src/promptflow/promptflow/_sdk/_configuration.py

```diff
@@ -158,4 +159,4 @@ def get_or_set_installation_id(self):
             return user_id
 
     def _to_dict(self):
-        return self.config
+        return self._config
```

Comment:
We have exported this config parameter, should we mark this parameter as public?

Reply:
it hasn't been introduced to public promptflow version yet, it's safe to make it private.